### ISSUE TITLE
add pervasive label errors tutorial as an examples notebook 

### ISSUE THREAD

**Issue by jwmueller** [cleanlab/examples#32]
Make examples notebook out of this:
https://github.com/cleanlab/label-errors/blob/main/examples/Tutorial%20-%20How%20To%20Find%20Label%20Errors%20With%20CleanLab.ipynb 

The code needs to be updated to use the latest version of cleanlab. Dataset files can stay where they are, but code to load them needs to be updated too. 

Consider replacing one of the existing examples/ notebooks with this one if they are quite similar.

- [ ] Also update the original notebook with link to the examples/ notebook stating:  
Here's how to find label issues in these datasets using the latest version of cleanlab: <link>

**Comment by jwmueller** [comment]
BTW the Imagenet predictions in that notebook are currently logits rather than predicted class probabilities, so need to apply softmax to them first to obtain `pred_probs` for use with cleanlab
